gromacs/config.py

```diff
@@ -637,10 +643,25 @@ def check_setup():
 check_setup()
 
 
-
 # Gromacs tools
 # -------------
 
+#: Runs GMXRC in a subprocess and put environment variables loaded by it in
+#: this environment.
+def set_gmxrc_environment(gmxrc):
+    envvars = ['GMXPREFIX', 'GMXBIN', 'GMXLDLIB', 'GMXMAN', 'GMXDATA',
+               'GROMACS_DIR', 'LD_LIBRARY_PATH', 'MANPATH', 'PKG_CONFIG_PATH',
+               'GROMACS_DIR', 'PATH']
+    cmdargs = ['bash', '-c', ". {0} && echo {1}".format(gmxrc,
+               ' '.join(['${0}'.format(v) for v in envvars]))]
+    try:
+        out = subprocess.check_output(cmdargs)
+        out = out.strip().split()
+        for key, value in zip(envvars, out):
+            os.environ[key] = value
+    except subprocess.CalledProcessError, OSError:
```

Comment:
in Py2 you need parentheses:
```python
except (subprocess.CalledProcessError, OSError):
```

otherwise the error trace will be assigned to the variable `OSError` and it will *not* trap `OSError`.